trucks.js

```diff
@@ -501,4 +502,168 @@ var foodTrucks = [
     Facebook: 'https://www.facebook.com/fulltilticecreams',
     Twitter: 'https://twitter.com/fticecream'
   }
-];
\ No newline at end of file
+];
+/* eslint-enable max-len */
+
+//function returns true if the currently checked truck is available
+//on the target day
+function filterBySchedule(truck, day) {
+  if (truck.schedule.indexOf(day) !== -1) {
+    return true;
+  } else {
+    return false;
+  }
+}
+ 
+//filters a list of trucks by their schedule and returns an array of
+//trucks (objects) that are available on a certain day
+var filterTrucksByDay = function(day){
+  var filteredTruckArray = foodTrucks.filter(function(truck){
```

Comment:
You can return the result of the filter call directly - no need to close the array and return it.